packages/material-ui/src/locale/index.ts

```diff
@@ -2,200 +2,242 @@ import * as React from 'react';
 import { ComponentsPropsList } from '../styles/props';
 
 export interface Localization {
```

Comment:
Oops I broke this https://github.com/mui-org/material-ui/pull/22293, they are fixed now 👍

Reply:
Does it mean we were missing a test?

Reply:
I thought argos would show us difference in the [locale example](https://next.material-ui.com/guides/localization/#main-content).

This is how it looks currently on the next branch:
![image](https://user-images.githubusercontent.com/4512430/91159885-fe266c00-e6c8-11ea-8912-9fe23b0a2a57.png)

And this is how it looks now in [this PR](https://deploy-preview-22347--material-ui.netlify.app/guides/localization/#locale-text):
![image](https://user-images.githubusercontent.com/4512430/91160028-32019180-e6c9-11ea-8a8a-96f50ed57a12.png)

Reply:
I'm afraid we don't run the visual regression under the guide:

https://github.com/mui-org/material-ui/blob/9ff98304e906ad1656d1a485293f7402b9534bce/test/regressions/index.js#L123

Reply:
But IMHO, visual regression should never be used when the same test can be written without. Explicit tests are often an order of magnitude faster to run and more reliable.

Reply:
This should definitely be covered with tests, I agree. My comment was just that I noticed it just by looking on the page, that's why my first thought was why this wasn't caught by argos. I will create an issue to track this, with the PR that broke it and we can discuss how we want to test this.